flake.nix

```diff
@@ -0,0 +1,78 @@
+{
+  
+  description = "Accessible tools for you to create, configure and build custom images of atomic Fedora distributions.";
```

Comment:
Bluebuild build these types of tools, yes, but cli by itself is not "tools". I'd rewrite this to "A commandline tool for creating, etc..."

Reply:
Oh LOL sorry I just copied the description from the bluebuild website, and I didnt see that this repo had a perfectly good description here